tests/test_router_ospf_area.rb

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) 2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/router_ospf'
+require_relative '../lib/cisco_node_utils/router_ospf_vrf'
+require_relative '../lib/cisco_node_utils/router_ospf_area'
+
+# TestRouterOspfArea - Minitest for RouterOspfArea node utility class
+class TestRouterOspfArea < CiscoTestCase
+  @skip_unless_supported = 'ospf_area'
+  @@pre_clean_needed = true # rubocop:disable Style/ClassVars
+
+  def setup
+    super
+    remove_all_ospfs if @@pre_clean_needed
+    @@pre_clean_needed = false # rubocop:disable Style/ClassVars
+  end
+
+  def teardown
+    remove_all_ospfs
+    super
+  end
+
+  def create_routerospfarea_default(router='Wolfpack', name='default',
+                                    area_id='1.1.1.1')
+    RouterOspfArea.new(router, name, area_id)
+  end
+
+  def create_routerospfarea_vrf(router='Wolfpack', name='blue',
+                                area_id='1.1.1.1')
```

Comment:
please look at the comment I gave for area_id integer above. I think we have some confusion about this PR. Can we discuss tomorrow as there are lot of comments here which I am not sure how to go about?